nodes/edge_detection.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+import sys
+import rospy
+
+import cozmo
+from cozmo.util import degrees
+
+from visualization_msgs.msg import Marker
+
+class CornerDetector(object):
+    """
+    A corner detector class that uses cozmo's sensor to map the dimensions of a
```

Comment:
Uses Cozmo's built in edge detector to find the corners of a rectangular surface